libuavcan/test/util/avl_tree.cpp

```diff
@@ -0,0 +1,610 @@
+/*
+ * Copyright (C) 2019 Theodoros Ntakouris <zarkopafilis@gmail.com>
```

Comment:
Added a coverage report. Looks pretty good. It'd be nice to cover some more of the branch conditions missed but overall this is excellent work. Thanks!

[covhtml.zip](https://github.com/UAVCAN/libuavcan/files/2918257/covhtml.zip)